src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/display/DisplayLine.kt

```diff
@@ -116,24 +115,40 @@ abstract class DisplayLine {
         this.backgroundColor = backgroundColor
     }
 
-    fun registerClicked(method: Any) = run {
+    fun registerClicked(method: Any) = apply {
```

Comment:
Changing these `run`s to `apply`s is a breaking change

Reply:
That is unfortunate because i feel like it should be possible to register triggers for each line before they get added to the Display. Without this change the trigger methods need to be registered after the line was added to the Display which I think is counter intuitive and is a pain especially when trying to use the `setLines` function.

Reply:
I'm not sure how this change would affect that, but regardless we cannot introduce breaking changes no matter how nice they may be.